change/@fluentui-react-components-6f24dd75-7d89-45d6-9fcd-c910eff6f642.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Moving Textarea to beta",
+  "packageName": "@fluentui/react-components",
+  "email": "email not defined",
```

Comment:
Please update your `user.email` git config to have a meaningful email in the change entries